discovery/consul/consul.go

```diff
@@ -113,9 +113,8 @@ type SDConfig struct {
 	// The list of services for which targets are discovered.
 	// Defaults to all services if empty.
 	Services []string `yaml:"services,omitempty"`
-	// An optional tag used to filter instances inside a service. A single tag is supported
-	// here to match the Consul API.
-	ServiceTag string `yaml:"tag,omitempty"`
+	// An optional tag used to filter instances inside a service.
```

Comment:
Is it an and or an or?

Reply:
Based on #4786 it's an `AND` on all the tags from here: https://www.consul.io/api/catalog.html#tag

I'll update the comment.